fix(showcase): creator detail 'worked on' site list multiple lines

riencoertjens opened the pull request:
<!--
  Have any questions? Check out the contributing docs at https://gatsby.dev/contribute, or
  ask in this Pull Request and a Gatsby maintainer will be happy to help :)
-->

## Description

<!-- Write a brief description of the changes introduced by this PR -->

## Related Issues

<!--
  Link to the issue that is fixed by this PR (if there is one)
  e.g. Fixes #1234, Addresses #1234, Related to #1234, etc.
-->

pieh commented:
Hey @riencoertjens, could you share before/after screenshot for the changes in `template-creator-details.js` ?

riencoertjens commented:
before: 
<img width="1508" alt="Screenshot 2019-04-19 at 18 59 29" src="https://user-images.githubusercontent.com/36336875/56434552-554a4e80-62d5-11e9-95a2-4340ffb1855f.png">

after:
<img width="1552" alt="Screenshot 2019-04-19 at 18 58 53" src="https://user-images.githubusercontent.com/36336875/56434551-554a4e80-62d5-11e9-8eb1-9ef53759db24.png">

pieh commented on docs/sites.yml:
```diff
@@ -5383,3 +5383,87 @@
+  categories:
+    - Athlete
+    - Blog
+    - Netlify CMS
```
I think we could skip specifying "Netlify CMS", given that source is not provided, so fact that Nelify CMS is used wouldn't be visible to showcase visitors?

pieh commented on docs/sites.yml:
```diff
@@ -5383,3 +5383,87 @@
+  categories:
+    - logopedics
+    - One Page
+    - Netlify CMS
```
Similar here

pieh commented on docs/sites.yml:
```diff
@@ -5383,3 +5383,87 @@
+  description: >
+    Cycling themed café coming this april in Sint Niklaas, Belgium. Has funky css-grid trickery!
+  categories:
+    - Café
```
Could we perhaps use Coffee here too? I know it's not exact, but given that Coffee is added in other entry we would end up with Café and Coffee categories each with single entry

pieh commented on docs/sites.yml:
```diff
@@ -5383,3 +5383,87 @@
+  url: https://www.hetgroeiatelier.be/
+  main_url: https://www.hetgroeiatelier.be/
+  description: >
+    Workspace for talent development and logopedics. onepage site with basic info and small calendar CMS.
```
```suggestion
    Workspace for talent development and logopedics. One page site with basic info and small calendar CMS.
```

pieh commented on docs/sites.yml:
```diff
@@ -5383,3 +5383,87 @@
+  description: >
+    Workspace for talent development and logopedics. onepage site with basic info and small calendar CMS.
+  categories:
+    - logopedics
```
This category seems very specific and likely to be one-off - ideally we could use something more generic that would apply to more sites to avoid categories that would hold just single entries

riencoertjens commented:
those comments all make sense, it was obviously a long day... 
I made those changes, added some source_urls and changed my mind about the my name 😅

riencoertjens commented:
@pieh have you had a chance to look at this?

pieh commented:
Sorry about delay @riencoertjens and thanks for patience.
